Cybereason/Cybereason.manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+    "author": "Cybereason",
+    "categories": [
+        "Compliance and Security",
+        "Sumo Logic Certified"
+    ],
+    "communityURL": "https://support.sumologic.com/hc/en-us/community/topics/200263058-Applications-and-Integrations",
+    "description": "The Cybereason app for Sumo Logic enables Security Operations teams to leverage the Cybereason Malop to detect and end attacks faster",
+    "helpURL": "https://nest.cybereason.com/",
```

Comment:
Should we point this help url to github Readme documentation? Or is there a web page also which is hosting the docs?

Reply:
Sure please point it to the Github Readme doc.

Reply:
Then please update the manifest url to https://github.com/SumoLogic/sumologic-public-partner-apps/tree/master/Cybereason

Reply:
nevermind I'll merge this PR now so we can deploy today. I'll update the url while creating the deploy PR.